main.go

```diff
@@ -40,13 +40,17 @@ const indexHTML = `<!DOCTYPE html>
     const pre = document.createElement('pre');
     pre.innerText = await resp.text();
     document.body.appendChild(pre);
-    return;
+  } else {
+    const src = await resp.arrayBuffer();
+    const go = new Go();
+    const result = await WebAssembly.instantiate(src, go.importObject);
+    go.argv = {{.Argv}};
+    go.run(result.instance);
+  }
+  const reload = await fetch('__wait');
```

Comment:
Add a comment `The server sends a response for '_wait' when a request is sent to '_notify'.`